src/lib/helpers.js

```diff
@@ -5,6 +5,10 @@ import { promisify } from 'util'
 import Toast from '../components/toast'
 // import { GivethBridge } from '@giveth/bridge-contract'
 
+export const isUserRegistered = user => {
+  return !!(user && user.name && user.email)
```

Comment:
@RamRamez 
I think it would be more readable if you change it to 
Boolean(user && user.name && user.email)

Reply:
@RamRamez 
I did it myself, hope you don't mind it